newsroom/wire/views.py

```diff
@@ -77,9 +77,13 @@
 HOME_EXTERNAL_ITEMS_CACHE_KEY = "home_external_items"
 
 
-async def set_permissions(wire_item: WireItem, ignore_latest=False):
+async def set_permissions(
+    wire_item: WireItem, ignore_latest: bool = False, service: Optional[WireSearchServiceAsync] = None
```

Comment:
With Python 3.10+ there is no need to import and use `typing.Optional`, you can instead use the in-built type `| None` which does the same without the need to import a type

Reply:
Done!